packages/aws-cdk-lib/aws-codepipeline/lib/pipeline.ts

```diff
@@ -373,6 +478,8 @@ export class Pipeline extends PipelineBase {
   private readonly enableKeyRotation?: boolean;
   private readonly reuseCrossRegionSupportStacks: boolean;
   private readonly codePipeline: CfnPipeline;
+  private readonly pipelineType?: PipelineType;
```

Comment:
To prevent changes from occurring with the existing version of the template, this parameter is made optional rather than overwritten by V1 when unspecified in props.